crates/aptos/src/move_tool/mod.rs

```diff
@@ -851,6 +852,18 @@ impl FunctionArgType {
             FunctionArgType::Hex => bcs::to_bytes(
                 &hex::decode(arg).map_err(|err| CliError::UnableToParse("hex", err.to_string()))?,
             ),
+            FunctionArgType::HexArray => {
+                let mut encoded = vec![];
+                for sub_arg in arg.split(',') {
+                    encoded.push(hex::decode(sub_arg).map_err(|err| {
+                        CliError::UnableToParse(
+                            "hex_array",
+                            format!("Failed to parse hex array: {:?}", err.to_string()),
```

Comment:
Can you explain more here, like "expected args separated by commas"

Reply:
Yes, I want to get this out, and then we need better documentation here in general around all the input types